interfaces/builtin/ppp.go

```diff
@@ -41,11 +41,6 @@ var pppConnectedPlugAppArmor = []byte(`
 @{PROC}/@{pid}/loginuid r,
 capability setgid,
 capability setuid,
-/sbin/resolvconf rix,
-/run/resolvconf** rw,
-/etc/resolvconf/** rw,
-/etc/resolvconf/update.d/* ix,
-/lib/resolvconf/* ix,
```

Comment:
Actually pppd shouldn't put resolvconf things into place, only network-manager should. If another component wants to use pppd and wants to put resolvconf items in place it should really use network-control to make this visible as adjusting resolvconf is quite critical for the system.